Makefile

```diff
@@ -16,8 +28,7 @@ LIBS			=
 # Compiler and linker settings
 NAME 			= 	plazza
 CXX				= 	g++
-CXXFLAGS		= 	-W -Wall -Wextra -std=c++20 -I./include  \
-				 	-L./libs
+CXXFLAGS		= 	-W -Wall -Wextra -std=c++20 -I./include -L./libs -luuid
```

Comment:
Can you move -luuid into LINUX_FLAGS it is not required in macOS and blocks compilation